apps/ledger-live-mobile/src/families/mina/SendRowsCustom.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import type { Account } from "@ledgerhq/types-live";
+import { Transaction as MinaTransaction } from "@ledgerhq/live-common/families/mina/types";
+import type { Transaction } from "@ledgerhq/live-common/generated/types";
+import SendRowMemo from "./SendRowMemo";
+import { BaseComposite, StackNavigatorProps } from "~/components/RootNavigator/types/helpers";
+import { SendFundsNavigatorStackParamList } from "~/components/RootNavigator/types/SendFundsNavigator";
+import { SignTransactionNavigatorParamList } from "~/components/RootNavigator/types/SignTransactionNavigator";
+import { SwapNavigatorParamList } from "~/components/RootNavigator/types/SwapNavigator";
+import { ScreenName } from "~/const";
+
+type Navigation = BaseComposite<
+  | StackNavigatorProps<SendFundsNavigatorStackParamList, ScreenName.SendSummary>
+  | StackNavigatorProps<SignTransactionNavigatorParamList, ScreenName.SignTransactionSummary>
+  | StackNavigatorProps<SwapNavigatorParamList, ScreenName.SwapSelectFees>
+>;
+
+type Props = {
+  transaction: Transaction;
+  account: Account;
+} & Navigation;
+export default function MinaSendRowsCustom(props: Props) {
+  const { transaction, ...rest } = props;
+  return (
+    <>
+      <SendRowMemo {...rest} transaction={transaction as MinaTransaction} />
```

Comment:
Why do we need a cast here ?

Reply:
Ahh, this change was bc of this failing typecheck, I try to find a fix...
![Screenshot 2025-03-25 at 16 05 19](https://github.com/user-attachments/assets/be97a3e7-1888-46c8-8b2a-22fdd3243b00)